### PR TITLE
add command lines and web server

### PR DESCRIPTION
**Pull request by mfdeveloper508**
add command lines, web server, tests

### CODE REVIEW

**Comment by adamdecaf** [comment]
Nice work! It matches the some of the other CLI setups you've created. Can we add an openapi.yaml to this?

**Comment by mfdeveloper508** [comment]
sure, I will add soon

File: api/api.yml [hunk 1]
```diff
@@ -0,0 +1,1572 @@
...
+                $ref: '#/components/schemas/ISOMessage'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ISOMessage'
+        '400':
+          description: bad request
+          content:
+            text/plain:
+              schema:
+                type: string
```
Comment (adamdecaf) [inline api/api.yml:506]: I think we'll want to use `.Problem(err)` from moov-io/base to render a JSON object with the error. Thoughts? 

https://github.com/moov-io/base/blob/master/http/server.go#L27

**Comment by adamdecaf** [comment]
The API spec looks good! Here's a preview 

https://editor.swagger.io/?url=https://raw.githubusercontent.com/moov-io/iso8583/add-cmd-server/api/api.yml

File: api/api.yml (hunk 1 above)
Comment (mfdeveloper508) [reply api/api.yml]: fixed

**Pull request merged**